returnn/torch/engine.py

```diff
@@ -521,7 +521,7 @@ def _load_optimizer(self, epoch):
         filename = self.get_epoch_model_filename(epoch=epoch - 1) + ".opt.pt"
         if os.path.isfile(filename):
             self._updater.load_optimizer(filename, device=self._device)
-        elif self.config.bool("use_fresh_optimizer_for_missing_checkpoint", False):
+        elif self.config.bool("allow_missing_optimizer", False):
```

Comment:
```suggestion
        elif self.config.bool("allow_missing_optimizer_checkpoint", False):
```

Now that I look at it, maybe we do better add the "checkpoint" to it, because otherwise it could mean you are really missing the optimizer itself.